eyebrowse.el

```diff
@@ -445,6 +451,7 @@ another appropriate window config."
     (when (and (> (length window-configs) 1)
                (or (not eyebrowse-close-window-config-prompt)
                    (yes-or-no-p "Close current window config?")))
+      (run-hooks 'eyebrowse-pre-window-delete-hook)
```

Comment:
It would make more sense to move that right before deletion is executed. As is the hook is run, a window config is switched to and only then the other one is deleted.

Reply:
Added hook just before deletion is invoked.